packages/manager/apps/hpc-vmware-managed-vcd/src/types/vcd-organization-backup.interface.ts

```diff
@@ -0,0 +1,36 @@
+export enum BackupResourceStatus {
+  CREATING = 'CREATING',
+  DISABLED = 'DISABLED',
+  DISABLING = 'DISABLING',
+  READY = 'READY',
+  REMOVED = 'REMOVED',
+  UPDATING = 'UPDATING',
+}
+
+interface BackupOffer {
```

Comment:
To be consistent with the used naming convention

```suggestion
interface IBackupOffer {
```